python/ray/experimental/state.py

```diff
@@ -388,12 +388,21 @@ def task_profiles(self):
     return task_info
 
   def dump_catapult_trace(self, path):
+    """Dump task profiling information to a file.
+
+    This information can be viewed as a timeline of profiling information by
+    going to chrome://tracing in the chrome web browser and loading the
+    appropriate file.
+
+    Args:
+      path: The filepath to dump the profiling information to.
+    """
```

Comment:
consider renaming it `dump_chrome_trace` (see `generate_chrome_trace_format` in TensorFlow); probably more people know chrome://tracing than know catapult